doc/Programming-guide/Key-platform-features/Data-collection/Oracle-NoSQL-log-appender/index.md

```diff
@@ -0,0 +1,214 @@
+```
+
+where file ```oracleNoSQLlogAppender.json``` contains following data:
+
+```
+{
+    "pluginClassName":"org.kaaproject.kaa.server.appenders.oraclenosql.appender.OracleNoSqlLogAppender",
+    "pluginTypeName":"Oracle NoSQL",
+    "applicationId":"5",
+    "applicationToken":"82635305199158071549",
+    "name":"Sample Oracle NoSQL log appender",
+    "description":"Sample Oracle NoSQL log appender",
+    "headerStructure":[
+        "KEYHASH",
+        "VERSION",
+        "TIMESTAMP",
+        "TOKEN",
+        "LSVERSION"
+    ],
+    "maxLogSchemaVersion":2147483647,
+    "minLogSchemaVersion":1,
+    "tenantId":"1",
+    "jsonConfiguration":"{\"storeName\":\"kvstore\",\"kvStoreNodes\":[{\"host\":\"localhost\",\"port\":5000}],\"username\":null,\"walletDir\":null,\"pwdFile\":null,\"securityFile\":null,\"transport\":null,\"ssl\":null,\"sslCipherSuites\":null,\"sslProtocols\":null,\"sslHostnameVerifier\":null,\"sslTrustStore\":null,\"sslTrustStoreType\":null}"
+}
+```
+
+Example result:
+
+```json
+{
+    "applicationId":"5",
+    "applicationToken":"82635305199158071549",
+    "confirmDelivery":true,
+    "createdTime":1466506070066,
+    "createdUsername":"devuser",
+    "description":"Sample Oracle NoSQL log appender",
+    "headerStructure":[
+        "KEYHASH",
+        "VERSION",
+        "TIMESTAMP",
+        "TOKEN",
+        "LSVERSION"
+    ],
+    "id":"163842",
+    "jsonConfiguration":"{\"storeName\":\"kvstore\",\"kvStoreNodes\":[{\"host\":\"localhost\",\"port\":5000}],\"username\":null,\"walletDir\":null,\"pwdFile\":null,\"securityFile\":null,\"transport\":null,\"ssl\":null,\"sslCipherSuites\":null,\"sslProtocols\":null,\"sslHostnameVerifier\":null,\"sslTrustStore\":null,\"sslTrustStoreType\":null}",
+    "maxLogSchemaVersion":2147483647,
+    "minLogSchemaVersion":1,
+    "name":"Sample Oracle NoSQL log appender",
+    "pluginClassName":"org.kaaproject.kaa.server.appenders.oraclenosql.appender.OracleNoSqlLogAppender",
+    "pluginTypeName":"Oracle NoSQL",
+    "tenantId":"1"
+}
+```
+
+# Playing with Oracle NoSQL log appender
+
+1. Download archive with [Oracle nosql database](http://www.oracle.com/technetwork/database/database-technologies/nosqldb/downloads/index.html)
+and install it to your Kaa server.
+2. Use [following](https://blogs.oracle.com/charlesLamb/entry/oracle_nosql_database_in_5) tutorial for more information about this database.
+3. Create an application using Admin UI or [Admin REST API]({{root_url}}Programming-guide/Server-REST-APIs #TODO).
+4. Add custom log schema that will be using for saving logs in database.<br/>
+![Add log schema](attach/nosql-log-appender1.png)
+5. Add Oracle Nosql log appender and define configuration for it.<br/>
+![Add log appender](attach/nosql-log-appender2.png)
+6. Generate SDK appropriate to your platform and write code to send logs to database.
```

Comment:
Generate Kaa SDK ~~appropriate to~~ **for** your platform and write code to send logs to database.